app/pages/1_Scenario_comparison.py

```diff
@@ -66,10 +207,13 @@ def main():
             network_map, option
         )
 
-    # TODO fix tech map to match with the Generators or Links
-    #df_techs = [tech_map[c] for c in df.columns]
-    #tech_colors = get_colors_map()
-    #plot_color = [tech_colors[c] for c in df_techs]
+    new_cols = pd.Index(tools.config["preferred_order"]).intersection(df.columns).append(
+        df.columns.difference(tools.config["preferred_order"])
+    )
+    df = df[new_cols].iloc[:,::-1]
```

Comment:
That is this `.iloc[:,::-1]` indexing which reverts the order. We may just drop it, if that is not an intended effect